### DISCUSSION TITLE
[SUGGESTION] Add diagrams to documentation.

### DISCUSSION THREAD

**Discussion by ash-dvlpr** [FirstGearGames/FishNet#573] [Documentation]
Today I found out that there's a [SceneManager Event Diagram.png](https://github.com/FirstGearGames/FishNet/blob/main/Assets/FishNet/Demos/SceneManager/SceneManager%20Event%20Diagram.png) on the Demos folder and thought that Diagrams like that could be really handy if available on the online docs.

You could add it to the [SceneManager Component Documentation](https://fish-networking.gitbook.io/docs/manual/components/managers/scenemanager) 

And another could be added to the [NetworkBehaviour Component](https://fish-networking.gitbook.io/docs/manual/components/network-behaviour-components) ilustrating the NetworkBehaviour event process, similar to [Unity's ExecutionOrder Documentation](https://docs.unity3d.com/Manual/ExecutionOrder.html).

**Comment by maxkratt** [comment]
There are some diagrams for those, they are in the Guides section though:
https://fish-networking.gitbook.io/docs/manual/guides/network-behaviour-guides
https://fish-networking.gitbook.io/docs/manual/guides/scene-management/scene-events

**Reply by ash-dvlpr** [reply]
Oh, thanks for the pointer because I didn't know that was on the "Guides" section; which I considered to be analagous to "Examples".

For example with the NetworkBehaviour:
- https://fish-networking.gitbook.io/docs/manual/components/network-behaviour-components
- https://fish-networking.gitbook.io/docs/manual/guides/network-behaviour-guides

It is a Component, yet most of the info on it is on another place, making it harder to find what you're looking for.

Would be nice if there was at least a hint on the [Component page](https://fish-networking.gitbook.io/docs/manual/components/network-behaviour-components) link telling you:
"Additional information can be found [here](https://fish-networking.gitbook.io/docs/manual/guides/network-behaviour-guides)".

Or something on those lines ^^

**Reply by maxkratt** [reply]
I think that's quite a good idea, I'll let the team know about it so it can hopefully be implemented. Thanks for the suggestion!

**Comment by mitsh** [comment]
yess. diagrams are really usefull! we need more diagrams. maybe we can do for PV2. that would be great!
![IMG_1282](https://github.com/FirstGearGames/FishNet/assets/895798/10c5a7e9-9240-46e1-bb7d-7c7fde37bf4d)

**Comment by FirstGearGames** [comment]
Added to the roadmap under a different thread requesting the same.
